packages/core/src/browser/widget-manager.ts

```diff
@@ -140,8 +167,10 @@ export class WidgetManager {
 
     /**
      * Get the widget for the given description.
+     * @param factoryId The widget factory id.
+     * @param options  The widget factory specific information.
```

Comment:
minor: additional extra whitespaces after the parameter.

Reply:
Fixed